cmd/kubeadm/app/phases/upgrade/postupgrade_test.go

```diff
@@ -40,14 +40,14 @@ func TestMoveFiles(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Failed to create cert file %s: %v", certPath, err)
 	}
-	defer certFile.Close()
+	certFile.Close()
```

Comment:
Are we not supposed to still close handles?

Reply:
We're closing them immediately after opening them, we don't need to defer anymore.